server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/model/IRVComparisonAuditTests.java

```diff
@@ -598,15 +734,27 @@ public void testComputeRecordDiscrepancyCVR_BACD_ACVR_ABCD(RecordType auditedTyp
     ca.recordDiscrepancy(auditInfo, 1);
 
     checkDiscrepancies(ca, 1, 0, 0, 0, 0);
+
+    riskMeasurementCheck(ca, List.of(Pair.make(10, 1.0),
```

Comment:
These risk measurement checks are great, and I think you should add a comment to say that you actually calculated them carefully to determine what they were supposed to be (maybe ref the equation in super simple and/or our design doc).